.eslintrc.js

```diff
@@ -68,6 +68,8 @@ module.exports = {
           "_selected_hover",
           "_selected_focus",
           "_selected_focusVisible",
+          "_selected_after",
+          "_focus_selected_after",
```

Comment:
observation: weird that you added four, but eslint only cared about two of them

Reply:
yeah that's why the error was throwing me off